test/components/submission/filters.spec.js

```diff
@@ -443,4 +443,21 @@ describe('SubmissionFilters', () => {
         odataRequests.should.equal(1);
       });
   });
+
+  describe('reset button', () => {
+    beforeEach(() => {
+      testData.extendedForms.createPast(1);
+    });
+
+    it('resets the filters when clicked', () =>
+      loadComponent('submitterId=1&submitterId=2')
```

Comment:
How about adding one more filter here? Like `&reviewState=null`. It'd be nice to see that multiple filters can get reset at once.